src/components/MessageBody.tsx

```diff
@@ -46,20 +73,78 @@ const useStyles = makeStyles((theme) => ({
     '& tr:nth-child(even)': {
       backgroundColor: 'lightgray',
     },
+    '& img': {
+      maxWidth: '100%',
+    },
   },
 }));
 
 type Props = {
   messageBody: string;
 };
 
+const renderCode = ({
+  inline,
+  className: classNameInit,
+  children: codeContent,
+  ...props
+}: CodeProps): ReactElement => {
+  const match = /language-(\w+)/.exec(classNameInit || '');
```

Comment:
Consider using [`String.prototype.match`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match) as [RegExp objects are stateful](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/exec). Obviously in this case this doesn't change anything, but this would clarify the intention for future maintenance.

Reply:
Ok thank you !